desktop/cmp/dash/canvas/impl/DashCanvasView.js

```diff
@@ -29,52 +29,55 @@ export const dashCanvasView = hoistCmp.factory({
     model: uses(DashCanvasViewModel, {publishMode: ModelPublishMode.LIMITED}),
 
     render({model, className}) {
-        const {viewSpec, viewState, containerModel, id, positionParams, title, ref} = model,
-            {extraMenuItems, contentLocked, renameLocked} = containerModel;
+        const {viewSpec, viewState, containerModel, id, positionParams, title, ref, hidePanelHeader} = model,
+            {extraMenuItems, contentLocked, renameLocked} = containerModel,
+            headerProps = hidePanelHeader ? {} : {
```

Comment:
consider making a component for the popover here -- will be less of a wall of code, and also more stable -- right now if the observables used in any of your menu items change you need to re-render this entire component!